roofit/roofitcore/src/RooProdPdf.cxx

```diff
@@ -766,22 +766,20 @@ Int_t RooProdPdf::getPartIntList(const RooArgSet* nset, const RooArgSet* iset, c
   RooArgSet *norm, *integ, *xdeps, *imps;
 
   // Group irriducible terms that need to be (partially) integrated together
-  RooLinkedList groupedList;
+  std::list<RooLinkedList> groupedList;
```

Comment:
`std::vector` almost always wins. Are you sorting?

Reply:
Is RooLinkedList necessary here? Most of the time the answer was "no", so try to remove it here.

Reply:
Ok, maybe not worth all the effort. This happens only once per fit, I assume.

Reply:
Elements are removed, that's why I stayed with the list

Reply:
There's still a good chance that the vector is faster, but I won't ask you to measure that, as it most likely doesn't make a difference for general RooFit usage.